src/openMVG/sfm/pipelines/localization/SfM_Localizer.hpp

```diff
@@ -25,13 +25,12 @@ struct Image_Localizer_Match_Data
             // found through the feature matching procedure.
   
   // pt2D and pt3D have the same number of columns.
-            
   std::vector<size_t> vec_inliers; // Index mask for both pt3D and pt2D whose elements
                                    // represent the column indices of inliers in  pt2D 
-                                   // and pt3D.   
+                                   // and pt3D.
   
   // Upper bound pixel(s) tolerance for residual errors
-  double error_max = std::numeric_limits<double>::infinity();
+  double error_max = 100;
```

Comment:
It's because of the ACRansac. If you put infinity the acransac will run in full a contrario mode. Otherwise it will firtst try a normal RANSAC until it found a model with enough support and only then it will switch to a contrario mode. It is somehow related to the bugfix of last week openMVG/openMVG#537